x-pack/plugins/security_solution/public/common/components/autocomplete/field_value_lists.test.tsx

```diff
@@ -8,14 +8,22 @@ import { ThemeProvider } from 'styled-components';
 import { mount } from 'enzyme';
 import euiLightVars from '@elastic/eui/dist/eui_theme_light.json';
 import { EuiComboBox, EuiComboBoxOptionOption } from '@elastic/eui';
+import { act } from 'react-dom/test-utils';
 
 import { getField } from '../../../../../../../src/plugins/data/common/index_patterns/fields/fields.mocks.ts';
 import { AutocompleteFieldListsComponent } from './field_value_lists';
 import { getFoundListSchemaMock } from '../../../../../lists/common/schemas/response/found_list_schema.mock';
+import { getListResponseMock } from '../../../../../lists/common/schemas/response/list_schema.mock';
+import { wait } from '../../../common/lib/helpers';
 
+jest.mock('../../../common/lib/kibana');
 const mockStart = jest.fn();
+const mockKeywordList = getListResponseMock();
+mockKeywordList.id = 'keyword_list';
+mockKeywordList.type = 'keyword';
+mockKeywordList.name = 'keyword list';
```

Comment:
nit, I have been using non-mutations like below:

```ts
mockKeywordList:ListSchema = { ...getListResponseMock(), id: 'keyword_list', type: 'keyword', name: 'keyword list' };
```

Even in tests to just keep things consistent. Might seem silly since `getListResponseMock()` already protects its self from the mutations, but constantly keeping non-mutating patterns in the code base is always a good idea for everyone to copy and paste as examples.

Reply:
That was kinda messy on my part, thanks for the suggestion! I updated it and will keep an eye out for it in tests.